### ISSUE TITLE
View Samples in Oculus GO "VR Mode" / Fullscreen (nor sure how to name it)

### ISSUE THREAD

**Issue by nkpm** [facebook/react-360#797]
### Description

I'm not sure if it's a bug or a misunderstanding on my side.

#### Expected behavior

Running the samples I would expect to be able on the oculus go browser: to open them in the VR mode. This means not seeing the content inside the browser an pan arround with the cursor but being inide the scene and moving arround by moving my head.

#### Actual behavior

I can only see the scene/video/photo inside the browser window and pan arround with the cursor / controller

### Reproduction

I installed and ran it on my macbook and point to it via http://mbp:8081/index.html
### Solution

none avaliable 

### Additional Information

* 'react-360' package version: react-360@1.1.0
* 'react-360-web' package version: react-360-web@1.1.0
* Operating System: running on a local macbook
* VR Device: oculus go

**Comment by jrjones** [comment]
This started when Oculus Browser was updated to 9.0 and WebVR support was removed.
https://developer.oculus.com/documentation/oculus-browser/browser-release-notes/

Oculus has provided [instructions for fixing this](https://developer.oculus.com/documentation/oculus-browser/browser-vr-xr/?locale=en_US), but I've not been able to get any of it to work with react-360. So far, I have tried forcing a new version of three.js in the node_modules version in addition to the Oculus recommendations to no avail. 

Unfortunately, I think we have to wait for the React-360 team to fix this, assuming there is still a React-360 team... but they've been radio silent for a while. After the COVID/zombie-apocalypse is over hopefully they will make the status of this project more clear.

**Comment by zicopan** [comment]
Yea React 360 doesn't support WebXR right now so the only way you'll be able to view React 360 content in VR is to use the Firefox Reality browser. Otherwise, consider using A-frame.

**Comment by BerendWeij** [comment]
@zicopan Are you able to view React 360 content in Firefox Reality? Perhaps I am using an older version of React 360, but I do not get a 'view in webvr' option in Firefox Reality.

**Comment by zicopan** [comment]
Yes, try viewing it via the iframe from the React 360 website at the bottom of [this](https://facebook.github.io/react-360/) page. I see the option to view in VR when in Firefox Reality, but not in Oculus Browser

**Comment by fxsegura** [comment]
@BerendWeij you might have to add some extra code that gives you the option to View in VR.
